client/index.jsx

```diff
@@ -1,5 +1,29 @@
+/* eslint-disable react/destructuring-assignment */
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
```

Comment:
Do we want to merge this change into the group master, or is this more for rendering your widget as you develop it_

Reply:
I'm not sure if I understand what you are asking but basically, I am trying to keep my commits small. I removed the import of the app because we don't have an app component. And I am making this pull so that the team can see my file structure.